src/Angor/Avalonia/AngorApp/Sections/Browse/Details/InvestmentDashboard.axaml

```diff
@@ -4,8 +4,7 @@
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
              xmlns:dt="clr-namespace:AngorApp.Sections.Browse.Details"
              xmlns:i="https://github.com/projektanker/icons.avalonia"
-             mc:Ignorable="d" d:DesignWidth="600"
-             MinWidth="300"
+             mc:Ignorable="d" d:DesignWidth="600" 
```

Comment:
I think we need to set a max width because on a large screen it doesn't look very good when it's stretched out.